src/cli/llm/infer_type_name.rs

```diff
@@ -76,20 +84,35 @@ impl InferTypeName {
         Self { wizard: Wizard::new(model, secret) }
     }
 
+    /// All generated type names starts with "GEN__"
+    #[inline]
+    fn is_auto_generated(type_name: &str) -> bool {
+        type_name.starts_with("GEN__")
```

Comment:
Let's not hard code it here. We should keep a const static which hold the prefix.